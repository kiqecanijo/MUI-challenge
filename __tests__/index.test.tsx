import { render, screen } from '@testing-library/react'
import App from '../pages/index'
import renderer from 'react-test-renderer'

describe('Index Page render ðŸ““', () => {
  it('Match snapshot ðŸ“·', () => {
    const tree = renderer.create(<App />).toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('Should contain a Button ðŸŸ¢', () => {
    render(<App />)
    const input = screen.getByText('Go to Contacts')
    expect(input).toBeInTheDocument()
  })
})
