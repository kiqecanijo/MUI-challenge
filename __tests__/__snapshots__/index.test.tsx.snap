// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index Page render ðŸ““ Match snapshot ðŸ“· 1`] = `
<div
  className="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
>
  <div
    className="MuiGrid-root css-176m31h-MuiGrid-root"
  >
    <a
      href="/contacts"
      onClick={[Function]}
      onMouseEnter={[Function]}
    >
      <button
        className="MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeLarge MuiButton-containedSizeLarge MuiButtonBase-root  css-jh47zj-MuiButtonBase-root-MuiButton-root"
        disabled={false}
        onBlur={[Function]}
        onContextMenu={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type="button"
      >
        Go to Contacts
      </button>
    </a>
  </div>
</div>
`;
